src/Life/Github.hs

```diff
@@ -52,13 +54,24 @@ pushka (Branch branch) commitMsg = do
     "git" ["push", "-u", "origin", branch]
 
 -- | Creates repository on GitHub inside given folder.
-createRepository :: Owner -> Repo -> IO ()
-createRepository (Owner owner) (Repo repo) = do
+createRepository :: Maybe Owner -> Repo -> IO ()
+createRepository mo (Repo repo) = do
+    owner <- case mo of
+          Nothing -> getUsername
+          Just (Owner o) -> pure o
     let description = ":computer: Configuration files"
     "git" ["init"]
     "hub" ["create", "-d", description, owner <> "/" <> repo]
     pushka master "Create the project"
 
+-- | Get current user name from the local global git config.
+getUsername :: IO Text
+getUsername = do
+    login <- "git" $| ["config", "user.name"]
```

Comment:
It should be `login`, not `name`. We're interested in the GitHub nickname. Unfortunately, it's not the same as `user.name` and there's no way to get info about Github nickname unless users specify it somewhere in the `.gitconfig`. In `hit` we decided to choose `user.login` field. You can easily configure this field by performing a command like:

```
git config --global user.login <your_login>
```
```suggestion
    login <- "git" $| ["config", "user.login"]
```